frame/wormhole/backing/s2s/src/lib.rs

```diff
@@ -81,6 +80,8 @@ pub mod pallet {
 		type RingPalletId: Get<PalletId>;
 		#[pallet::constant]
 		type RingLockMaxLimit: Get<RingBalance<Self>>;
+		#[pallet::constant]
+		type BlocksPerDay: Get<BlockNumberFor<Self>>;
```

Comment:
Can we calculate this from block interval?

Reply:
Yes this number is calculated by `sencodsPerDay(60*60*24s)/interval(6s)=14_400`
Do you mean the interval is a dynamic value, and we should calculate when used?

Reply:
I'm not so sure whether you can get that value here: 

https://github.com/darwinia-network/darwinia/blob/2f9f86141821c3bb89a10cb18d96dfec5e6aa9d7/runtime/crab/src/constants.rs#L46

Reply:
This is just the value here needed https://github.com/darwinia-network/darwinia/blob/2f9f86141821c3bb89a10cb18d96dfec5e6aa9d7/runtime/crab/src/constants.rs#L63